node/operations.js

```diff
@@ -22,6 +22,8 @@
 
 var errors = require('./errors.js');
 
+var CONN_STALE_PERIOD = 1500;
```

Comment:
A little worried about magic number. We might have to fiddle with this in production.